pkg/cmd/server/start/start_allinone.go

```diff
@@ -20,22 +20,18 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 
 	"github.com/openshift/origin/pkg/cmd/server/admin"
-	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 	"github.com/openshift/origin/pkg/cmd/server/start/kubernetes"
 	cmdutil "github.com/openshift/origin/pkg/cmd/util"
 )
 
 type AllInOneOptions struct {
-	MasterArgs *MasterArgs
-	NodeArgs   *NodeArgs
-
-	CreateCerts      bool
-	ConfigDir        util.StringFlag
-	MasterConfigFile string
-	NodeConfigFile   string
-	PrintIP          bool
-	Output           io.Writer
-	DisabledFeatures []string
+	MasterOptions
```

Comment:
No objection

On Aug 13, 2015, at 9:02 AM, David Eads <notifications@github.com> wrote:

In pkg/cmd/server/start/start_allinone.go
<https://github.com/openshift/origin/pull/4132#discussion_r36970911>:

>  	"github.com/openshift/origin/pkg/cmd/server/start/kubernetes"
>  	cmdutil "github.com/openshift/origin/pkg/cmd/util"
>  )
>
>  type AllInOneOptions struct {
> -	MasterArgs *MasterArgs
> -	NodeArgs   *NodeArgs
> -
> -	CreateCerts      bool
> -	ConfigDir        util.StringFlag
> -	MasterConfigFile string
> -	NodeConfigFile   string
> -	PrintIP          bool
> -	Output           io.Writer
> -	DisabledFeatures []string
> +	MasterOptions

I really dislike anonymous inclusion because of the unexpected interface
fulfillment and name hiding. Do you object to giving this a name?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4132/files#r36970911>.